airesources/CSharp/rungame.bat

```diff
@@ -1,2 +1,4 @@
-dotnet build
-./halite -d "240 160" "dotnet ./bin/Debug/netcoreapp1.1/csharp2.dll" "dotnet ./bin/Debug/netcoreapp1.1/csharp2.dll"
\ No newline at end of file
+@echo off
+
+dotnet build -c Release
+halite -d "240 160" "dotnet ./bin/Release/netcoreapp2.0/Halite2.dll" "dotnet ./bin/Debug/Release/Halite2.dll"
```

Comment:
Does "/Debug/Release/" in path make sense? Last line.

Reply:
Fixed.